lib/Dashboard/TalkWidget.php

```diff
@@ -50,12 +54,21 @@ class TalkWidget implements IAPIWidget, IIconWidget, IButtonWidget, IOptionWidge
 	protected MessageParser $messageParser;
 
 	public function __construct(
+		IUserSession $userSession,
+		Config $talkConfig,
 		IURLGenerator $url,
 		IL10N $l10n,
 		Manager $manager,
 		ParticipantService $participantService,
 		MessageParser $messageParser
 	) {
+		$user = $userSession->getUser();
+		if ($user instanceof IUser && $talkConfig->isDisabledForUser($user)) {
+			// This is dirty and will log everytime a user opens the dashboard or requests the api,
+			// so we should look for a different solution in the server.
```

Comment:
```suggestion
			// @todo This is dirty and will log everytime a user opens the dashboard or requests the api,
			// so we should look for a different solution in the server.
```

Reply:
PR with Api is already there, so once it is reviewed an in we can immediately drop this here again: https://github.com/nextcloud/server/pull/34832

Reply:
Is the action to remove this is a to-do, not?

Reply:
Yes should be, but is adding the `@todo` worth the energy to run 2h of CI?
You could instead approve the server PR and if daniel agrees as well we have the 2 thumbs up and we can merge the server PR and I can make the follow up PR to use the correct interface instead of commiting the todo here?